Tests/configure_and_test_integration_instances.py

```diff
@@ -143,7 +143,8 @@ def configure_integration_instance(integration, client, prints_manager):
                                  print_color, 0, LOG_COLORS.GREEN)
     prints_manager.execute_thread_prints(0)
     integration_instance_name = integration.get('instance_name', '')
-    integration_params = integration.get('params')
+    changes_map = {'%%SERVER_HOST%%': server_host}
```

Comment:
I see you added the param server_host and this is the only use for this in this integration (same is true in the set_integration_params method).

If the idea is that the change_map can grow in the future with additional mappings, it might be wiser to create the change_map in the main function and send it instead of the server_host specifically - I think it would be easier to expand if needed.